tests/e2e_test/manifest_reconciliation_test.go

```diff
@@ -0,0 +1,115 @@
+package e2e_test
+
+import (
+	"github.com/kyma-project/lifecycle-manager/api/shared"
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	declarativev2 "github.com/kyma-project/lifecycle-manager/internal/declarative/v2"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	. "github.com/kyma-project/lifecycle-manager/pkg/testutils"
+)
+
+var _ = Describe("Manifest Skip Reconciliation Label", Ordered, func() {
+	kyma := NewKymaWithSyncLabel("kyma-sample", "kcp-system", v1beta2.DefaultChannel,
+		v1beta2.SyncStrategyLocalSecret)
+	module := NewTemplateOperator(v1beta2.DefaultChannel)
+
+	InitEmptyKymaBeforeAll(kyma)
+	CleanupKymaAfterAll(kyma)
+
+	Context("Given kyma deployed in KCP", func() {
+		It("When enabling Template Operator", func() {
+			Eventually(EnableModule).
+				WithContext(ctx).
+				WithArguments(runtimeClient, defaultRemoteKymaName, remoteNamespace, module).
+				Should(Succeed())
+			By("Then the Module Operator is deployed on the SKR cluster")
+			Eventually(CheckIfExists).
+				WithContext(ctx).
+				WithArguments("template-operator-controller-manager", "template-operator-system", "apps", "v1",
+					"Deployment", runtimeClient).
+				Should(Succeed())
+			By("And the SKR Module Default CR is in a \"Ready\" State")
+			Eventually(CheckSampleCRIsInState).
+				WithContext(ctx).
+				WithArguments("sample-yaml", "kyma-system", runtimeClient, "Ready").
+				Should(Succeed())
+			By("And the KCP Kyma CR is in a \"Ready\" State")
+			Eventually(KymaIsInState).
+				WithContext(ctx).
+				WithArguments(kyma.GetName(), kyma.GetNamespace(), controlPlaneClient, shared.StateReady).
+				Should(Succeed())
+		})
+
+		It("When the Manifest is labelled to skip reconciliation", func() {
+			labels, err := GetManifestLabels(ctx, kyma.GetName(), kyma.GetNamespace(), module.Name, controlPlaneClient)
+			Expect(err).ToNot(HaveOccurred())
+			labels[declarativev2.SkipReconcileLabel] = "true"
+			err = SetManifestLabels(ctx, kyma.GetName(), kyma.GetNamespace(), module.Name, controlPlaneClient, labels)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("When deleting the SKR Default CR")
+			/*err = SetFinalizer("sample-yaml", "kyma-system", "operator.kyma-project.io",
```

Comment:
Please remove this comment out code

Reply:
im not finished with the test. It has a peculiar behaviour in the local testing. I'll let you know when it's ready for review